12-rest/classworks/Alexey_Goremykin/server.js

```diff
@@ -0,0 +1,98 @@
+const Koa = require('koa');
+const Router = require('koa-router');
+const bodyParser = require('koa-bodyparser');
+const jwt = require('jsonwebtoken');
+const koajwt = require('koa-jwt');
+const axios = require('axios');
+const fs = require('fs');
+
+const app = new Koa();
+const publicRouter = new Router();
+const privateRouter = new Router();
+const secret = 'ololo';
+let token = '';
+
+const db = {};
+
+app.use(bodyParser())
+
+publicRouter.post('/signup', (ctx, next) => {
+  let user = ctx.request.body.login;
+  db[user] = ctx.request.body.password;
```

Comment:
also you should check that there is no user with such login already. otherwise the user can be rewritten by any other user sending /signup with the same login